test/pytest/test_data/dlrm_model.py

```diff
@@ -0,0 +1,42 @@
+from dataclasses import dataclass
+from typing import List
+
+from dlrm_factory import DLRMFactory
+from torchrec.datasets.criteo import DEFAULT_CAT_NAMES, DEFAULT_INT_NAMES
+
+
+def simple_dlrm_model_config():
+    """
+    This simplified configuration is used during unittesting.
+    """
+
+    @dataclass
+    class DLRMModelConfig:
```

Comment:
There's some code duplication here, let's define `DLRMModelConfig` in one place

Reply:
Moved it to a separate file